.github/workflows/action.yml

```diff
@@ -292,3 +292,39 @@ jobs:
         update: true
         install: base-devel git
     - run: git describe --dirty --tags
+
+  errorhandling:
+    runs-on: windows-latest
```

Comment:
Please, add `needs: []` to execute each job sequentially. Alternatively, move these additional jobs to the beginning, so the order of the execution view matches the source.

Reply:
I was wondering why things get run sequentially. Why is that needed/wanted?

Reply:
There is a limit of 20 concurrent jobs (I'm not sure it's per workflow or per organization). Apart from that, when cache is being tested, concurrent jobs might fight for it. Last, when this was unoptimized, all the jobs fetching packages from MSYS2 at the same time made things quite slower.

We might relax the requirement if you want, and execute several at a time. The current fully sequential approach is extremely conservative.

Reply:
Since this is used for "testing" I would try to test it how it is used in the "real world". But I'll add a needs for the cache part.

Reply:
The point is that in the "real world" most users will use one of these jobs only. We are mostly testing multiple syntaxes to do the same, or to test all available options. Hence, in this sense, doing it sequential is the most "real". The jobs which are defined through a matrix are already executed in parallel. So, the case of doing something on MINGW32 and MINGW64 "at the same time" is already covered.

Reply:
ok, fair enough, I added the `need: []`